src/components/registration/FormFieldGroup.tsx

```diff
@@ -46,9 +46,10 @@ const FormFieldGroup = ({ fields, onChange }: FormFieldGroupProps) => {
               "aria-label": `${field.role} ${field.name}`,
             }}
           >
-            {field.question_type === QuestionTypes.MULTIPLE_CHOICE && (
-              <MenuItem value="none">None</MenuItem>
```

Comment:
we should probably leave an empty value in here so that users can "unselect" if they want to. could we leave the "None" option here with `value=""`?